src/components/form/form.component.js

```diff
@@ -30,11 +37,10 @@ const Form = ({
   ...rest
 }) => {
   const [isFooterSticky, setIsFooterSticky] = useState(false);
-
+  const sidebarContext = useContext(SidebarContext);
```

Comment:
If you provide a default value to the `SidebarContext` you can destructure `isInSidebar` instead here